### PR TITLE
Updated the Photon JITM message to test for better results

### PR DESCRIPTION
**Pull request by jessefriedman**
@richardmuscat and I want to test if a new message will increase activations of Photon from the just-in-time-message.

### CODE REVIEW

File: class.jetpack-jitm.php [hunk 1]
```diff
@@ -43,7 +43,7 @@ function photon_msg() {
 					</svg>
 				</div>
 				<p>
-					<?php _e( 'Deliver super-fast images to your visitors that are automatically optimized for any device.', 'jetpack' ); ?>
+					<?php _e( 'Speed up your media library and save bandwidth costs by using a free content network.', 'jetpack' ); ?>
```
Comment (dereksmart) [inline class.jetpack-jitm.php]: s/content network/content delivery network?

Comment (richardmuscat) [inline class.jetpack-jitm.php]: @dereksmart: do you mean it should be "free content *delivery* network"?

File: class.jetpack-jitm.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ function photon_msg() {
 					</svg>
 				</div>
 				<p>
-					<?php _e( 'Deliver super-fast images to your visitors that are automatically optimized for any device.', 'jetpack' ); ?>
+					<?php _e( 'Speed up your media library and save bandwidth costs by using a free content delivery network.', 'jetpack' ); ?>
```
Comment (jeherve) [inline class.jetpack-jitm.php]: About "Speed up your media library": Photon isn't applied to the Media Library, though, it's only applied to images inserted into posts.

Comment (richardmuscat) [inline class.jetpack-jitm.php]: I was trying to go for a message that's within the context of what the user is doing working under the assumption that in most cases stuff in the media library is to be used in posts and pages.

cc @jessefriedman

**Comment by dereksmart** [comment]
@jessefriedman @richardmuscat -- updated verbiage, please merge if it looks good to you.

**Comment by dereksmart** [comment]
talked with @jessefriedman on slack -- said it's good

**Pull request merged**